README.Rmd

```diff
@@ -63,5 +63,15 @@ knitr::opts_chunk$set(
 
 Check out all the [codemetar man](https://docs.ropensci.org/codemetar/articles/index.html) for tutorials on other cool stuff you can do with codemeta and json-ld.  
 
+```{r child='man/rmdhunks/minimeta.Rmd'} 
+```
+
+```{r addscript, results="asis", echo=FALSE}
```

Comment:
Is this the correct way to add the file to the homepage @cboettig?

Reply:
looks good to me